va/va_dec_av1.h

```diff
@@ -0,0 +1,546 @@
+    uint8_t                 feature_mask[8];
+
+    /** \brief Reserved bytes for future use, must be zero */
+    uint32_t                va_reserved[VA_PADDING_LOW];
+
+} VASegmentationStructAV1;
+
+/** \brief Film Grain Information
+  */
+typedef struct _VAFilmGrainStructAV1 {
+    union {
+        struct {
+            /** \brief Specify whether or not film grain is applied on current frame.
+             *  If set to 0, all the rest parameters should be set to zero
+             *  and ignored.
+             */
+            uint32_t        apply_grain                                 : 1;
+            uint32_t        chroma_scaling_from_luma                    : 1;
+            uint32_t        grain_scaling_minus_8                       : 2;
+            uint32_t        ar_coeff_lag                                : 2;
+            uint32_t        ar_coeff_shift_minus_6                      : 2;
+            uint32_t        grain_scale_shift                           : 2;
+            uint32_t        overlap_flag                                : 1;
+            uint32_t        clip_to_restricted_range                    : 1;
+            uint32_t        reserved                                    : 20;
+        } bits;
+        uint32_t            value;
+    } film_grain_info_fields;
+
+    uint16_t                random_seed;
+    /*  value range [0..14] */
+    uint8_t                 num_y_points;
+    uint8_t                 point_y_value[14];
+    uint8_t                 point_y_scaling[14];
+    /*  value range [0..10] */
+    uint8_t                 num_cb_points;
+    uint8_t                 point_cb_value[10];
+    uint8_t                 point_cb_scaling[10];
+    /*  value range [0..10] */
+    uint8_t                 num_cr_points;
+    uint8_t                 point_cr_value[10];
+    uint8_t                 point_cr_scaling[10];
+    /*  value range [-128..127] */
+    int8_t                  ar_coeffs_y[24];
+    int8_t                  ar_coeffs_cb[25];
+    int8_t                  ar_coeffs_cr[25];
+    uint8_t                 cb_mult;
+    uint8_t                 cb_luma_mult;
+    /*  value range [0..512] */
+    uint8_t                 cb_offset;
+    uint8_t                 cr_mult;
+    uint8_t                 cr_luma_mult;
+    /*  value range [0..512] */
+    uint8_t                 cr_offset;
+
+    /** \brief Reserved bytes for future use, must be zero */
+    uint32_t                va_reserved[VA_PADDING_LOW];
+
+} VAFilmGrainStructAV1;
+
+typedef enum {
+    /** identity transformation, 0-parameter */
+    VAIDENTITY    = 0,
+    /** translational motion 2-parameter */
+    VATRANSLATION = 1,
+    /** simplified affine with rotation + zoom only, 4-parameter */
+    VAROTZOOM     = 2,
+    /** affine, 6-parameter */
+    VAAFFINE      = 3,
+    VATRANS_TYPES,
+} VATransformationType;
+
+typedef struct _VAWarpedMotionParamsAV1{
+    VATransformationType    wmtype;
+    int32_t                 wmmat[8];

```

Comment:
It's no wmmat in spec. So how I fill these fields? What is the mean of 8?
If possible, please keep the name in spec.
If not possible, it's better to have some document point to spec.

Reply:
@ZefuLI, any feedback for this?

thanks

Reply:
wm[7].wmmat[8] is corresponding to gm_params[ ref ][ j ] ( for ref = LAST_FRAME..ALTREF_FRAME, for j = 0..5.)  in spec, for wmmat[8], valid index value is [0..5], we use 8 here is to keep consistent with libaom.
Meanwhile, I will add commment to point to spec.

Reply:
thanks